#Request
int64 NUM_STEP
---
#Response
bool result
